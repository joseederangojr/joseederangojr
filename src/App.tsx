import React from 'react'
import tw from 'twin.macro'
import styled from '@emotion/styled'
import { motion } from 'framer-motion'

const Greeting = styled(motion.h1)`
  ${tw`
  text-6xl font-extrabold
  tracking-widest uppercase
  text-center my-auto
  text-gray-100 my-auto
  block
`}
`

function App() {
  return (
    <div tw="w-screen h-screen bg-gray-800 flex">
      <Greeting initial={{ opacity: 0, y: 500 }} animate={{ opacity: 1, y: 0 }}>
        React + TailwindCSS + Framer Motion + Emotion
      </Greeting>
    </div>
  )
}

export default App
