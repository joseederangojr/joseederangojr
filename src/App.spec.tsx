import React from 'react'
import { render } from '@testing-library/react'
import App from 'App'

it('Will render App without crashing', () => {
  const tree = render(<App />)
  expect(tree.asFragment()).toMatchSnapshot()
})
