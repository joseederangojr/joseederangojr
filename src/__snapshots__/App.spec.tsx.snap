// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Will render App without crashing 1`] = `
<DocumentFragment>
  <div
    class="css-nzfevl-Container ea7idt90"
  >
    <img
      class="css-m1c6nw-Image ea7idt93"
      height="300"
      src="https://scontent.fmnl4-1.fna.fbcdn.net/v/t1.0-9/76695227_419986712238950_4227379487311921152_o.jpg?_nc_cat=103&ccb=2&_nc_sid=09cbfe&_nc_ohc=pxkB-cEi5HEAX-HSMG-&_nc_ht=scontent.fmnl4-1.fna&oh=3850f5505fe83b2876fe07ffdf5e8d0d&oe=5FC5C39D"
      style="opacity: 0; transform: scale(0) rotate(360deg) translateZ(0);"
      width="300"
    />
    <h1
      class="css-j558b2-Text ea7idt91"
      style="opacity: 0; transform: translateY(250px) translateZ(0);"
    >
      Jose Ederango Jr.
    </h1>
    <h1
      class="css-1tqlpd1-Text-SubText ea7idt92"
      style="opacity: 0; transform: translateY(50px) translateZ(0);"
    >
      Building quality software
    </h1>
    <div
      class="css-1737amr-IconList ea7idt95"
    >
      <a
        class="css-ot2quv-Link ea7idt94"
        href="https://facebook.com/joseederangojr"
        target="_blank"
        title="Facebook"
      >
        <svg
          fill="none"
          height="44"
          stroke="#f7fafc"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1"
          viewBox="0 0 24 24"
          width="44"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
            stroke="none"
          />
          <path
            d="M7 10v4h3v7h4v-7h3l1 -4h-4v-2a1 1 0 0 1 1 -1h3v-4h-3a5 5 0 0 0 -5 5v2h-3"
          />
        </svg>
      </a>
      <a
        class="css-ot2quv-Link ea7idt94"
        href="https://twitter.com/joseederangojr"
        target="_blank"
        title="Twitter"
      >
        <svg
          fill="none"
          height="44"
          stroke="#f7fafc"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1"
          viewBox="0 0 24 24"
          width="44"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
            stroke="none"
          />
          <path
            d="M22 4.01c-1 .49 -1.98 .689 -3 .99c-1.121 -1.265 -2.783 -1.335 -4.38 -.737s-2.643 2.06 -2.62 3.737v1c-3.245 .083 -6.135 -1.395 -8 -4c0 0 -4.182 7.433 4 11c-1.872 1.247 -3.739 2.088 -6 2c3.308 1.803 6.913 2.423 10.034 1.517c3.58 -1.04 6.522 -3.723 7.651 -7.742a13.84 13.84 0 0 0 .497 -3.753c-.002 -.249 1.51 -2.772 1.818 -4.013z"
          />
        </svg>
      </a>
      <a
        class="css-ot2quv-Link ea7idt94"
        href="https://github.com/joseederangojr"
        target="_blank"
        title="Github"
      >
        <svg
          fill="none"
          height="44"
          stroke="#f7fafc"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1"
          viewBox="0 0 24 24"
          width="44"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
            stroke="none"
          />
          <path
            d="M9 19c-4.3 1.4 -4.3 -2.5 -6 -3m12 5v-3.5c0 -1 .1 -1.4 -.5 -2c2.8 -.3 5.5 -1.4 5.5 -6a4.6 4.6 0 0 0 -1.3 -3.2a4.2 4.2 0 0 0 -.1 -3.2s-1.1 -.3 -3.5 1.3a12.3 12.3 0 0 0 -6.2 0c-2.4 -1.6 -3.5 -1.3 -3.5 -1.3a4.2 4.2 0 0 0 -.1 3.2a4.6 4.6 0 0 0 -1.3 3.2c0 4.6 2.7 5.7 5.5 6c-.6 .6 -.6 1.2 -.5 2v3.5"
          />
        </svg>
      </a>
      <a
        class="css-ot2quv-Link ea7idt94"
        href="https://linkedin.com/in/joseederangojr"
        target="_blank"
        title="LinkedIn"
      >
        <svg
          fill="none"
          height="44"
          stroke="#f7fafc"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1"
          viewBox="0 0 24 24"
          width="44"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
            stroke="none"
          />
          <rect
            height="16"
            rx="2"
            width="16"
            x="4"
            y="4"
          />
          <line
            x1="8"
            x2="8"
            y1="11"
            y2="16"
          />
          <line
            x1="8"
            x2="8"
            y1="8"
            y2="8.01"
          />
          <line
            x1="12"
            x2="12"
            y1="16"
            y2="11"
          />
          <path
            d="M16 16v-3a2 2 0 0 0 -4 0"
          />
        </svg>
      </a>
      <a
        class="css-ot2quv-Link ea7idt94"
        href="mailto:joseederangojr@gmail.com"
        target="_blank"
        title="Email"
      >
        <svg
          fill="none"
          height="44"
          stroke="#f7fafc"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1"
          viewBox="0 0 24 24"
          width="44"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
            stroke="none"
          />
          <circle
            cx="12"
            cy="12"
            r="4"
          />
          <path
            d="M16 12v1.5a2.5 2.5 0 0 0 5 0v-1.5a9 9 0 1 0 -5.5 8.28"
          />
        </svg>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
